core/src/test/java/uk/gov/gchq/koryphe/util/CustomObj.java

```diff
@@ -15,13 +15,31 @@
  */
 package uk.gov.gchq.koryphe.util;
 
+import com.fasterxml.jackson.annotation.JsonGetter;
+import com.fasterxml.jackson.annotation.JsonSetter;
+
 public final class CustomObj implements Comparable<CustomObj> {
-    public final String value = "1";
+    public final static String value = "1";
+
+    @JsonGetter
```

Comment:
Isn't `@JsonGetter` verbose if the method matches "getX" and same for setter?

Reply:
Yes, and no. I've made this change because SpotBugs complains that the field `value` should be a static constant and is unused (i.e. not accessed). The change to make the field static fixes one error, the change to add the explicit getter fixes the other. I felt that this was preferable to adding lots of specific exclusions into the `spotbugs-exclude.xml` file.